Use published images in chatbot-rag-app example (#395)
elastic/elasticsearch-labs: example-apps/chatbot-rag-app/README.md

### Run with docker

Docker compose is the easiest way, as you get one-step to:
* ingest data into elasticsearch
* run the app, which listens on http://localhost:4000

**Double-check you have a `.env` file with all your variables set first!**

```bash

> docker compose up --force-recreate

Author: I considered `--pull always` for reproducibility but figured the simpler command is better

Reviewer: agreed in principal, but `--force-recreate` is already there, too. I think we should do `--pull-always` until the telemetry stuff settles down, so that a future blog won't result in someone retrying and not seeing the new stuff. I know it may be overly cautious, so not hard opinion on this.

Author: Yeah given possible telemetry churn makes sense, added it

See [Langchain documentation][loader-docs] for more ways to load documents.

### Building from source with docker

> To build the app with Docker instead of using published images, pass the `--build`

Reviewer: ```suggestion
To build the app from source instead of using published images, pass the `--build`
```

> flag to Compose.

Reviewer: ```suggestion
flag to Docker Compose.
```
